src/toil_scripts/lib/urls.py

```diff
@@ -43,6 +42,19 @@ def download_url_job(job, url, name=None, s3_key_path=None, cghub_key_path=None)
     return job.fileStore.writeGlobalFile(fpath)
 
 
+def _download_from_genetorrent(url, file_path, cghub_key_path):
```

Comment:
While we're touching this function: it should be named `_download_with_genetorrent` or `_download_from_gnos`